addons/web/static/src/views/list/list_controller.js

```diff
@@ -621,11 +621,7 @@ export class ListController extends Component {
         return true;
     }
 
-    onSavedMulti(records) {
-        records.forEach((record) => {
-            record.selected = false;
-        });
-    }
+    onSavedMulti(records) {}
```

Comment:
fun fact: I think this was meant to be overridden in document (document_list_controller.js). But the method we override is `onSavedMultiRecords` which has a single match in the whole codebase (same for `onWillSaveMultiRecords`). Could you open an enterprise PR to remove them?